### ISSUE TITLE
Use archive.org for hyperlinks

### ISSUE THREAD

**Issue by macterra** [proftomwbell/Ulex#18]
The hyperlinks in the footnotes all mention a visited date, e.g. "Available at: https://www.law.cornell.edu/ucc/1 (visited April 30, 2016)." 

We can link directly to a snapshot of the online document through archive.org, e.g. https://web.archive.org/web/20160409073957/https://www.law.cornell.edu/ucc/1 is the most recent snapshot of the same document before April 30, 2016.

**Comment by proftomwbell** [comment]
Yes, that sounds very sensible.  The "(visited . . . )" usage comes from an academic setting and is not necessary here.
